grimoiressg/utils/__init__.py

```diff
@@ -0,0 +1 @@
+from .files import to_relative
```

Comment:
This is more correct:

```suggestion
from .files import to_relative as to_relative
```

Reply:
Oh, what's the difference?